File: ncDados/dOrdemServico.vb
Namespace nsOrdemServico

    Public Class ColecaoOrdemServico
        Inherits List(Of dOrdemServico)
    End Class

    Public Class dOrdemServico

        Private _cid As Nullable(Of Integer)
        Private _clienteid As Nullable(Of Integer)
        Private _veiculoid As Nullable(Of Integer)
        Private _observacoes As String
        Private _emissao As String
        Private _vendedor As String
        Private _loja As String
        Private _situacao As String

        Public Property cid() As Nullable(Of Integer)
            Get
                Return _cid
            End Get
            Set(ByVal value As Nullable(Of Integer))
                _cid = value
            End Set
        End Property

        Public Property clienteid() As Nullable(Of Integer)
            Get
                Return _clienteid
            End Get
            Set(ByVal value As Nullable(Of Integer))
                _clienteid = value
            End Set
        End Property

        Public Property veiculoid() As Nullable(Of Integer)
            Get
                Return _veiculoid
            End Get
            Set(ByVal value As Nullable(Of Integer))
                _veiculoid = value
            End Set
        End Property

        Public Property observacoes() As String
            Get
                Return _observacoes
            End Get
            Set(ByVal value As String)
                _observacoes = value
            End Set
        End Property

        Public Property emissao() As String
            Get
                Return _emissao
            End Get
            Set(ByVal value As String)
                _emissao = value
            End Set
        End Property

        Public Property vendedor() As String
            Get
                Return _vendedor
            End Get
            Set(ByVal value As String)
                _vendedor = value
            End Set
        End Property

        Public Property loja() As String
            Get
                Return _loja
            End Get
            Set(ByVal value As String)
                _loja = value
            End Set
        End Property

        Public Property situacao() As String
            Get
                Return _situacao
            End Get
            Set(ByVal value As String)
                _situacao = value
            End Set
        End Property

    End Class

End Namespace

File: ncModelos/dOrdemServico.cs
using System;
using System.Collections.Generic;

namespace nsOrdemServico
{
    public class ColecaoOrdemServico : List<dOrdemServico>
    {
    }

    public class dOrdemServico
    {
        private int? _cid;
        private int? _clienteid;
        private int? _veiculoid;
        private string _observacoes;
        private string _emissao;
        private string _vendedor;
        private string _loja;
        private string _situacao;

        public int? cid
        {
            get => _cid;
            set => _cid = value;
        }

        public int? clienteid
        {
            get => _clienteid;
            set => _clienteid = value;
        }

        public int? veiculoid
        {
            get => _veiculoid;
            set => _veiculoid = value;
        }

        public string observacoes
        {
            get => _observacoes;
            set => _observacoes = value;
        }

        public string emissao
        {
            get => _emissao;
            set => _emissao = value;
        }

        public string vendedor
        {
            get => _vendedor;
            set => _vendedor = value;
        }

        public string loja
        {
            get => _loja;
            set => _loja = value;
        }

        public string situacao
        {
            get => _situacao;
            set => _situacao = value;
        }
    }
}
